packages/sdk/wallets/wallet-hsm-azure/src/azure-key-vault-client.ts

```diff
@@ -144,6 +161,15 @@ export class AzureKeyVaultClient {
     }
   }
 
+  private async getKeyCurve(keyName: string): Promise<string> {
+    const key = await this.getKey(keyName)
+    if (key.key === undefined || key.key.crv === undefined) {
```

Comment:
This is my biggest unknown. It looks like the package doesn't give us any assurances through the typing. It looks like both the key property on the base key, or the crv property on the key can be undefined -- I don't know when / how that happens :(